src/Symfony/Bundle/DebugBundle/DependencyInjection/DebugExtension.php

```diff
@@ -41,6 +41,9 @@ public function load(array $configs, ContainerBuilder $container)
             ->addMethodCall('setMinDepth', array($config['min_depth']))
             ->addMethodCall('setMaxString', array($config['max_string_length']));
 
+        $container->getDefinition('var_dumper.html_dumper')
+            ->addMethodCall('setTheme', array($config['theme']));
```

Comment:
setTheme is 4.2 - but the bundle accepts v4.1. We should ignore the option when var-dumper 4.1 is installed.